src/utilities/hydration.js

```diff
@@ -2,8 +2,9 @@ export const rehydrate = function(obj) {
   try {
     return JSON.parse(new Buffer(obj, 'base64').toString());
   } catch (e) {
-    alert('Unable to parse values passed in url query parameters');
-    console.error(e);
+    if (typeof alert !== 'undefined') {
```

Comment:
Add an else block to this statement where you default to using `console.error`.  Code that fails silently is no bueno ;)